spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/RewriteManifestsSparkAction.java

```diff
@@ -133,15 +143,17 @@ public RewriteManifestsSparkAction rewriteIf(Predicate<ManifestFile> newPredicat
 
   @Override
   public RewriteManifestsSparkAction stagingLocation(String newStagingLocation) {
-    this.stagingLocation = newStagingLocation;
+    if (shouldStageManifests) {
+      this.outputLocation = newStagingLocation;
+    } else {
+      LOG.warn("Ignoring provided staging location as new manifests will be committed directly");
+    }
     return this;
   }
 
   @Override
   public RewriteManifests.Result execute() {
-    String desc =
-        String.format(
-            "Rewriting manifests (staging location=%s) of %s", stagingLocation, table.name());
+    String desc = String.format("Rewriting manifests in %s", table.name());
```

Comment:
It was my bad to show the staging location in UI as it is really long and makes it hard to read. I doubt anyone sets this value (it is not even exposed via the procedure), so I removed it. Instead, we will see the table name properly.